src/rvwmo.adoc

```diff
@@ -278,52 +278,52 @@ _preserved program order_.
 
 The complete definition of preserved program order is as follows (and
 note that AMOs are simultaneously both loads and stores): memory
-operation l^A^ precedes memory operation _b_ in
+operation _a_ precedes memory operation _b_ in
 preserved program order (and hence also in the global memory order) if
-l^A^ precedes _b_ in program order,
-l^A^ and _b_ both access regular main memory
+_a_ precedes _b_ in program order,
+_a_ and _b_ both access regular main memory
 (rather than I/O regions), and any of the following hold:
 
-[[overlapping-orering]]
+[[overlapping-ordering]]
 * Overlapping-Address Orderings:
 . _b_ is a store, and
-l^A^ and _b_ access overlapping memory addresses
-. l^A^ and _b_ are loads,
-_x_ is a byte read by both l^A^ and
+_a_ and _b_ access overlapping memory addresses
+. _a_ and _b_ are loads,
+_x_ is a byte read by both _a_ and
 _b_, there is no store to _x_ between
-l^A^ and _b_ in program order, and
-l^A^ and _b_ return values for _x_
+_a_ and _b_ in program order, and
+_a_ and _b_ return values for _x_
 written by different memory operations
-. l^A^ is
+. _a_ is
 generated by an AMO or SC instruction, _b_ is a load, and
-_b_ returns a value written by l^A^
+_b_ returns a value written by _a_
 * Explicit Synchronization
 . There is a FENCE instruction that
```

Comment:
@elisa-riscv when you get a chance, can you please help me figure out how to make this numbering continuous from 1 through 13?

Reply:
Pushed a commit to add [start=n]...thanks!